src/main/java/teammates/common/datatransfer/InstructorAttributes.java

```diff
@@ -311,4 +311,20 @@ public boolean isEqualToAnotherInstructor(InstructorAttributes instructor) {
                    && this.privileges.equals(instructor.privileges);
         }
     }
+    
+    public boolean isCustomRole() {
+        return role.equals(Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_CUSTOM);
```

Comment:
Will `role` ever be `null`?
Reversing it may be more safe.
Calling `equals()` on a null `role` will cause a NPE, but reversing them will make it only return `false`.